fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryPlanWithIndex.java

```diff
@@ -84,10 +85,10 @@
     @Override
     default PlannerGraph rewritePlannerGraph(@Nonnull final List<? extends PlannerGraph> childGraphs) {
```

Comment:
I think that all of the implementations of `RecordQueryPlanWithIndex` override this method, so we can delete the default implementation on this interface.

Reply:
this is done differently now as we covering index scans are dealt with as a composition. please take another look.